advanced/ticket-templates.rst

```diff
@@ -4,21 +4,40 @@ Ticket Templates
 ================
 
 If you find yourself creating lots of tickets with the same basic attributes,
-use **ticket templates** to fill them in next time with a single click.
+use **ticket templates** to fill them in with a single click next time.
 
-.. figure:: /images/advanced/ticket-templates.jpg
-   :alt: Ticket template dialog
+.. figure:: /images/advanced/ticket-templates.png
+   :alt: Ticket template selection in new ticket dialog
    :align: center
 
-   Use the ticket pane to load or create ticket templates.
+   Use the ticket pane to load ticket templates.
 
-To create a new template, start by filling in a new ticket as usual.
-Then, instead of submitting the ticket form,
-use the **template dialog** in the ticket pane
-to enter a name for your new template, and click “Save”.
+On any *new ticket* dialogue use the 🗒️ tab on the right to display the
+*Templates* column. If Zammad has any templates for you to apply, you'll
+be provided with a drop down list to choose from.
 
-The next time you create a ticket,
-you’ll find your new template in the “Select template” dropdown.
-Click it to apply your saved attributes to the new ticket.
+Select a fitting template and press *Apply*.
+The configured ticket fields will be populated with the data from the template.
 
-.. hint:: Any template created by any agent will be available to **all** agents.
+.. warning::
+
+   With version 5.3 Zammad is able to detect "dirty fields".
```

Comment:
`dirty fields` is a quite technical term. I'm not sure if this is suitable for end user documentation. What about `field collisions` instead?